### ISSUE TITLE
Can't upload files via MediaHubField 

### ISSUE THREAD

**Issue by marcelosantos89** [outl1ne/nova-media-hub#27]
Hello,
I'm not sure if this is intended or not. But there is no way to upload files directly from the resource.
I'm using version 1.4.1 and this is what I see (https://imgur.com/a/VSpBP82)
Can anyone confirm if this is intended ?

**Comment by Tarpsvo** [comment]
That's weird. There should be Nova's file upload field on the left there. What Nova version are you using?

**Comment by marcelosantos89** [comment]
I'm using 4.11.0

**Comment by marcelosantos89** [comment]
Ok updated it to 4.19.3 and it shows the upload and dropzone.

**Comment by Tarpsvo** [comment]
Yeah, I think it's a discrepancy between Nova versions. I have to look into making it backwards compatible. Ie extarct the dropzone out of Nova just in case.

**Comment by marcelosantos89** [comment]
Thank you
